src/status_im2/contexts/chat/lightbox/top_view.cljs

```diff
@@ -6,10 +6,13 @@
     [react-native.orientation :as orientation]
     [react-native.platform :as platform]
     [react-native.reanimated :as reanimated]
+    [status-im.utils.http :as http]
```

Comment:
😢

Reply:
@J-Son89 can you explain? :)

Reply:
we're using status_im code in status_im2 namespaces. Not your issue and way beyond the scope of this pr! I think the team should put some effort into continuing/finishing the migration to status_im2 after the MVP. 🙏

Reply:
Oh the `http/replace-port` method was implemented/updated by @flexsurfer in a PR yesterday